src/App.jsx

```diff
@@ -14,33 +17,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const FIELD_ALPHA = 'alphabet';
+const FIELD_LENGTH = 'length';
```

Comment:
```suggestion
enum SortType {
  ALPHA = 'alphabet',
  LENGTH = 'length',
  NONE = ''
} 

```

Reply:
whats enum?

Reply:
Is this something from typescript? I haven't passed it yet

Reply:
Can i use constant instead of enum?